src/color/creating_reading.js

```diff
@@ -420,35 +424,36 @@ p5.prototype.hue = function(c) {
  *
  */
 
-p5.prototype.lerpColor = function(c1, c2, amt) {
+p5.prototype.lerpColor = function() {
```

Comment:
Sorry, I think this was mistake while I was cleaning up some stuff. I will double-check for other ones too!